src/containers/Transactions/test/actions.test.js

```diff
@@ -9,6 +11,8 @@ import OfferCreateData from './mock_data/rippledOfferCreate.json';
 import { formatTransaction } from '../../../rippled/lib/utils';
 
 describe('Transaction actions', () => {
+  jest.setTimeout(20000);
```

Comment:
What's this for?

Reply:
I added it because I wasn't sure if I just needed to add some more time for it to work. I left it in for now, but it'll be removed whenever I resolve all the issues.